### DISCUSSION TITLE
How to use utilities?

### DISCUSSION THREAD

**Discussion by djtobia** [russhughes/gc9a01_mpy#48] [General]
I am not quite sure how to run any of the util commands? I don't even know where exactly to run them? I have installed the uf2 onto my pico and can get the screen working, but now I want to convert an image from png to bitmap, like in the example here: https://russhughes.github.io/gc9a01_mpy/utils/image_converter.html

**Comment by pwolfamv** [comment]
They are python scripts.  After installing python you should be able to run the scripts via shell or command prompt.

**Reply by djtobia** [reply]
I've tried using the shell on thonny while connected to the pico, but it always says syntax error?

![image](https://github.com/user-attachments/assets/989abfbb-9dc5-4fec-bcee-e14cc5c33f82)

**Reply by russhughes** [reply]
The image_converter requires too many resources to run under MicroPython. The image_converter and other utilities must run on your PC instead of the MicroPython device.

**Reply by djtobia** [reply]
Ah, well that makes much more sense.
